src/components/User/CheckIn.jsx

```diff
@@ -36,8 +36,12 @@ const CheckinPage = () => {
           <p className="text-base">{session.user.email}</p>
         </div>
         <div className="bg-white w-2/3 h-1/3 lg:h-5/6 flex justify-center items-center flex-col rounded-lg m-auto">
-          <QRCodeSVG value={session.user.id} className="w-2/3 h-2/3" />
-          {date.toLocaleTimeString("en-US", { hour12: true })}
+          <QRCodeSVG
+            value={`${session.user.id}&${date.toLocaleTimeString("en-US", {
```

Comment:
toISOString

Reply:
gotcha. you wanted both date & time right? like the pic above?